### DISCUSSION TITLE
2.1.X版本docker-compose方法安装失败问题

### DISCUSSION THREAD

**Discussion by Odimmsun** [milvus-io/milvus#24262] [Q&A and General discussion]
根据https://milvus.io/docs/v2.1.x/install_standalone-docker.md这个地址安装milvus2.1.X版本，执行sudo docker-compose up -d命令后，报如下的错：
```
ERROR: The Compose file './docker-compose.yml' is invalid because:
networks.default value Additional properties are not allowed ('name' was unexpected)
```
版本信息：
$ docker --version
Docker version 20.10.21, build 20.10.21-0ubuntu1~18.04.3
$ docker-compose -version
docker-compose version 1.29.2, build unknown

**Comment by yhmo** [comment]
networks.default value Additional properties are not allowed ('name' was unexpected)
我觉得是你的docker版本太旧。我的docker是23.0.1可以正常起动
